reverbePrintemps/spiced
# More beautiful buttons + hover states

# Context

This PR aims to tackle the [the button styles challenge](https://github.com/spiced-academy/honey-web-dev/blob/main/sessions/css-basics/challenges-css-basics.md#button-styles).

# Changes

- [x] Styled buttons
- [x] Added hover states

# Test

1. Clone repo
2. Checkout branch
3. Run `index.html` in the browser
4. Witness beautiful buttons :)
5. Optional: hover over them to confirm hover states

# TODO


# GIF

![](https://media.giphy.com/media/l2JdTTKnnuY1IA1qM/giphy.gif)
